Packs/OracleCloudInfrastructure/Integrations/OracleCloudInfrastructureEventCollector/README.md

```diff
@@ -0,0 +1,46 @@
+|-------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|--------------|
+| Tenancy OCID                                                              | OCID of your tenancy. To get the value, see [Required Keys and OCIDs](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#Required_Keys_and_OCIDs).                                                 | True         |
+| User OCID                                                       | OCID of the user calling the API. To get the value, see [Required Keys and OCIDs](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#Required_Keys_and_OCIDs). <br> Example: ocid1.user.oc1..<unique_ID> | True        |
+| API Key Fingerprint                                                                   | Fingerprint for the public key that was added to this user. To get the value, see [Required Keys and OCIDs](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#Required_Keys_and_OCIDs).                                                                  | True        |
+| Private Key                                                                 | Private Key for authentication. <br> Important: The key pair must be in PEM format. For instructions on generating a key pair in PEM format, see [Required Keys and OCIDs](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#Required_Keys_and_OCIDs).                                                                       | True         |
+| Region                                               | An Oracle Cloud Infrastructure region. See [Regions and Availability Domains](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm#top). <br> Example: us-ashburn-1           | True         |
+| First fetch time    | First fetch time (< number > < time unit >, e.g., 12 hours, 1 day, 3 months). Default is 3 days. | False |
+| Trust any certificate (not secure) | Use SSL secure connection or ‘None’.  | False  |
+| User system proxy settings  | Runs the integration instance using the proxy server (HTTP or HTTPS) that you defined in the server configuration.  | False  |
+
+4. Click **Test** to validate the URLs, tokens, and connection.
+## Commands
+You can execute the following command from the Cortex XSOAR CLI, as part of an automation, or in a playbook. After you successfully execute a command, a DBot message appears in the War Room with the command details.
+
+#### oracle-cloud-infrastructure-get-events
+***
+Manual command to fetch and display events.
+
+#### Base Command
+
+`oracle-cloud-infrastructure-get-events`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| should_push_events | Set this argument to True in order to create events, otherwise the command will only display them. Default is False. | True | 
```

Comment:
```suggestion
| should_push_events | Set this argument to true in order to create events, otherwise the command will only display them. Default is false. | True | 
```